src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -79,7 +115,17 @@ public static DatasetModel populateDatasetModelFromDataset(Dataset dataset) {
                 .defaultProfileId(dataset.getDefaultProfileId().toString())
                 .createdDate(dataset.getCreatedDate().toString())
                 .schema(datasetSpecificationModelFromDatasetSchema(dataset))
-                .dataProject(dataset.getProjectResource().getGoogleProjectId());
+                .dataProject(dataset.getProjectResource().getGoogleProjectId())
+                .storage(storageResourceModelFromDatasetSummary(dataset.getDatasetSummary()));
+    }
+
+    private static List<StorageResourceModel> storageResourceModelFromDatasetSummary(DatasetSummary datasetSummary) {
+        return datasetSummary.getStorage().stream().map(storage ->
+            new StorageResourceModel()
+                .cloudResource(storage.getCloudResource())
+                .region(storage.getRegion())
+                .cloudPlatform(storage.getCloudPlatform()))
```

Comment:
I think this conversion of a `StorageResource` to a `StorageResourceModel` is done in a few places. You could remove some code duplication (and make this code clearer) by adding a method to do this to `StorageResource`, e.g. `StorageResource.toModel()`.